libarchive/archive_write_set_format_iso9660.c

```diff
@@ -2553,7 +2553,7 @@ set_date_time(unsigned char *p, time_t t)
 static void
 set_date_time_null(unsigned char *p)
 {
-	memset(p, '0', 16);
+	memset(p, (int)'0', 16);
```

Comment:
I agree that Coverity is probably being overly aggressive here, but an explicit marker to indicate that "yes, we really did mean '0' here" is low cost.